spec/jobs/users/follow_job_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe Users::FollowJob, type: :job do
+  include_examples "#enqueues_job", "users_follow", [1, 2, "User"]
+
+  describe "#perform_now" do
+    let(:user) { create(:user) }
+    let(:followable) { create(:user) }
+    let(:invalid_id) { -1 }
+
+    context "when followable doesn't exist" do
+      it "doesn't follow user" do
+        expect do
+          described_class.perform_now(user.id, invalid_id, followable.class.name)
+        end.to change(Follow, :count).by(0)
```

Comment:
Btw, you can use `not_to change(Follow, :count)` instead of checking that it was changed by 0.